### PR TITLE
Chibios colibri

### PR DESCRIPTION
**Pull request by peabody124**
Now that #1477 has merged, it would be good to update Colibri to use ChibiOS as
part of the upgrading of all the targets to ChibiOS. This requires more flash space
than is currently allocated for the board.

I opted to remove some components not used by most Colibri users rather than
increase the bootloader, but am open to change.

### CODE REVIEW

**Comment by peabody124** [comment]
@tbs-trappy can you comment on this? See this link https://github.com/peabody124/TauLabs/commit/175781ee48354f6c8515b29cd44386f70eeee97a to see the set of things I disabled.

**Comment by tbs-trappy** [comment]
Looks good to me. No issues with disabling these. How does this translate
to the V2?

**Comment by elafargue** [comment]
Removed most of the Fixed Wing path followers that make no sense for Gemini, but it looks like it preserves GPS navigation if you want to. @peabody124 you confirm ?

**Comment by peabody124** [comment]
Yeah GPS navigation should still be doable. PicoC is gone which is a nice feature but probably not used by Colibri users

File: flight/targets/colibri/fw/board.h [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+/*
+ * Setup for STMicroelectronics STM32F4-Discovery board.
+ */
+
+/*
+ * Board identifier.
+ */
+#define BOARD_QUANTEC_QUANTON
+#define BOARD_NAME                  "quanton flight control"
```
Comment (BrainFPV) [inline flight/targets/colibri/fw/board.h]: Should be Colibri?

Comment (elafargue) [inline flight/targets/colibri/fw/board.h]: Good point!

**Comment by elafargue** [comment]
Will fly-test ASAP & report

**Comment by elafargue** [comment]
Flight-tested: could not see a real difference in behavior of the airframe.

**Pull request merged**